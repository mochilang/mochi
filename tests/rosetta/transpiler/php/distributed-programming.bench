{
  "duration_us": 19,
  "memory_bytes": 160,
  "name": "main"
}