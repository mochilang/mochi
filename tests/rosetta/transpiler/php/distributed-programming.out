tax on 300 cents is 15 cents
{
  "duration_us": 19,
  "memory_bytes": 160,
  "name": "main"
}
